cfme/configure/access_control.py

```diff
@@ -249,3 +249,12 @@ def delete(self):
         tb_select('Delete this Role', invokes_alert=True)
         sel.handle_alert()
         flash.assert_success_message('Role "%s": Delete successful' % self.name)
+
+    def copy(self):
+        sel.force_navigate("cfg_accesscontrol_role_ed", context={"role": self})
+        tb.select('Configuration', 'Copy this Role to a new Role')
+        new_role = Role(name=self.name + "copy")
```

Comment:
I think we should have the ability here to "name" the new role, so by default it can take the ```self.name``` and append ```copy```, but we should be able to specify a name.